build.gradle

```diff
@@ -184,6 +184,8 @@ dependencies {
 
   implementation 'org.springframework.boot:spring-boot-starter-data-jpa:2.5.5'
   implementation('org.postgresql:postgresql')
+    implementation 'junit:junit:4.13.1'
```

Comment:
(all the junit 4 ones)

Reply:
So this was because I wanted to @ignore an integration test until Postgres was fixed. turns out @ignore is the junit 4 equivalent of @disabled - should now be fixed :)